### PR TITLE
Adding additional config parameters

### PR DESCRIPTION
**Pull request by s7clarke10**
There are some additional parameters for extra features added to this tap. This request is to make the additional config available.

All Singer definitions are stored in `/_data/taps/` or `/_data/targets`. The minimal requirement for adding a tap or target will match the following format:

```yaml
description: General description of what the company behind the API does
entity_type: Review the JSON Schema for latest options. Currently api, file, database.
entity_url: URL of the developer documentation or website
label: Properly formatted label of the connector
name: The unique name of the connector
singer_name: The Singer specific name of the connector. Typically tap-<name> or target-<name>
type: tap or target
variants: # an array of variant types
- default: true or false
  maintenance_status: "Options: active, beta, development, inactive, unknown"
  meltano_sdk: true or false
  name: Name of the GitHub/GitLab namespace
  pip_url: git+<git_url>.git or pip instal name
  repo: repo URL
  capabilities:
    - catalog
    - discover
  settings:
    # Can be an empty array if unknown: []
    # Describe the list of supported settings, for example:
    - name: project_ids
      kind: array
      label: Project IDs
      description: Array of project IDs.
    - name: username
      label: Username
      description: Credentials used when for connecting to the source.
    - name: password
      kind: password
      label: Password
      description: Credentials used when for connecting to the source.
```

## Checklist

- [ ] Add/update the file in the appropriate folder (`/taps` or `/targets`). The name of the file should match the name of the tap. If there is already one, add a descriptor to the name such as `-search`.
- [ ] Add/update the PNG logo image in `/assets/logos/<taps or targets>`. The image name must match the YAML file name.
- [ ] Tag `@tayloramurphy` or `@pnadolny13` to flag it for review. Or post to the [#hub](https://meltano.slack.com/archives/C01UGBSJNG5) channel on Meltano slack.

## Reviewer Checklist

- [ ] Validate file against JSON Schema. https://www.jsonschema.net/home is an option.
- [ ] Build website locally and validate everything works.

### CODE REVIEW

File: _data/meltano/extractors/tap-mssql/wintersrd.yml [hunk 1]
```diff
@@ -33,8 +33,15 @@ settings:
   description: The MSSQL port, defaulted to 1433.
 - name: tds_version
   label: tds_version
-  description: The TDS version of the connection. Defaults to 8.0, connecting to older
+  description: The TDS version of the connection. Defaults to 7.3, connecting to older
     databases (ex MS SQL 2005) requires 7.0 or earlier
+- name: use_date_datatype
+  kind: boolean
+  label: Use Date Datatype.
+  description: To emit a date datatype as a date without a time component or time without an UTC offset.
```
Comment (tayloramurphy) [inline _data/meltano/extractors/tap-mssql/wintersrd.yml:41]: @s7clarke10 does this have a default set by the tap? Given that it's a boolean I would guess false? Making it clear in the description would help on this one.

File: _data/meltano/extractors/tap-mssql/wintersrd.yml [hunk 2]
```diff
@@ -33,8 +33,15 @@ settings:
...
+- name: characterset:
```
Comment (tayloramurphy) [inline _data/meltano/extractors/tap-mssql/wintersrd.yml:42]:
```suggestion
- name: characterset
```

**Review by tayloramurphy — CHANGES_REQUESTED** [review]

File: _data/meltano/extractors/tap-mssql/wintersrd.yml (hunk 1 above)
Comment (s7clarke10) [reply _data/meltano/extractors/tap-mssql/wintersrd.yml:43]: Agreed, I have updated the config with a clearer explanation.

**Review by s7clarke10 — COMMENTED** [review]
Agreed, thanks for your feedback. I have updated the config file as per your suggestions.

File: _data/meltano/extractors/tap-mssql/wintersrd.yml [hunk 3]
```diff
@@ -33,8 +33,21 @@ settings:
...
+- name: use_date_datatype
+  kind: boolean
+  value: false
+  label: Use Date Datatype.
+  description: To emit a date datatype as a date without a time component or time without an UTC offset.
+    If this boolean config item is not set, the default behaviour is false i.e. emit date datatypes as a datetime.
+    It is recommended to set this on if you have time datatypes and are having issues uploading into into a target database.
+- name: characterset
+  label: Characterset
+  description: The characterset for the database / source system. The default is utf8, however older databases might use a charactersets like cp1252 for the encoding.
```
Comment (tayloramurphy) [inline _data/meltano/extractors/tap-mssql/wintersrd.yml:48]:
```suggestion
  description: |
    The characterset for the database / source system. The default is utf8, however older databases might use a charactersets like cp1252 for the encoding.
```

File: _data/meltano/extractors/tap-mssql/wintersrd.yml [hunk 4]
```diff
@@ -33,8 +33,22 @@ settings:
...
-  description: The TDS version of the connection. Defaults to 8.0, connecting to older
-    databases (ex MS SQL 2005) requires 7.0 or earlier
+  description: The TDS version of the connection. Defaults to 7.3, connecting to older
+    databases (ex MS SQL 2005) requires 7.0 or earlier. Please see the pymssql or FreeTSD documentation for more details.
+    Also note, while this is a confusing topic version 7.3 is more current than 8.0.
+- name: use_date_datatype
+  kind: boolean
+  value: false
+  label: Use Date Datatype.
+  description: To emit a date datatype as a date without a time component or time without an UTC offset.
```
Comment (tayloramurphy) [inline _data/meltano/extractors/tap-mssql/wintersrd.yml:43]:
```suggestion
  description: |
    To emit a date datatype as a date without a time component or time without an UTC offset.
```

File: _data/meltano/extractors/tap-mssql/wintersrd.yml [hunk 5]
```diff
@@ -33,8 +33,22 @@ settings:
   description: The MSSQL port, defaulted to 1433.
 - name: tds_version
   label: tds_version
-  description: The TDS version of the connection. Defaults to 8.0, connecting to older
-    databases (ex MS SQL 2005) requires 7.0 or earlier
+  description: The TDS version of the connection. Defaults to 7.3, connecting to older
```
Comment (tayloramurphy) [inline _data/meltano/extractors/tap-mssql/wintersrd.yml:36]:
```suggestion
  description: |
    The TDS version of the connection. Defaults to 7.3, connecting to older
```

File: _data/meltano/extractors/tap-mssql/wintersrd.yml [hunk 6]
```diff
@@ -33,8 +33,24 @@ settings:
   description: The MSSQL port, defaulted to 1433.
 - name: tds_version
   label: tds_version
```
Comment (tayloramurphy) [inline _data/meltano/extractors/tap-mssql/wintersrd.yml:35]:
```suggestion
  label: TDS Version
```

File: _data/meltano/extractors/tap-mssql/wintersrd.yml [hunk 7]
```diff
@@ -33,8 +33,24 @@ settings:
...
-  description: The TDS version of the connection. Defaults to 8.0, connecting to older
-    databases (ex MS SQL 2005) requires 7.0 or earlier
+  description: |
+    The TDS version of the connection. Defaults to 7.3, connecting to older
+    databases (ex MS SQL 2005) requires 7.0 or earlier. Please see the pymssql or FreeTSD documentation for more details.
+    Also note, while this is a confusing topic version 7.3 is more current than 8.0.
+- name: use_date_datatype
+  kind: boolean
+  value: false
+  label: Use Date Datatype.
```
Comment (tayloramurphy) [inline _data/meltano/extractors/tap-mssql/wintersrd.yml:43]:
```suggestion
  label: Use Date Datatype
```

**Review by tayloramurphy — APPROVED** [review]

**Pull request merged**